WordPress/Classes/Services/FollowCommentsService.swift

```
@@ -94,6 +94,26 @@ class FollowCommentsService: NSObject {
         }
     }
 
+    /// Toggles the notification setting for a specified post.
+    ///
+    /// - Parameters:
+    ///   - isNotificationsEnabled: Determines whether the user should receive notifications for new comments on the specified post.
+    ///   - success: Block called after the operation completes successfully.
+    ///   - failure: Block called when the operation fails.
+    @objc func toggleNotificationSettings(_ isNotificationsEnabled: Bool,
+                                          success: @escaping () -> Void,
+                                          failure: @escaping (Error?) -> Void) {
+        remote.updateNotificationSettingsForPost(with: postID, siteID: siteID, receiveNotifications: isNotificationsEnabled) { [weak self] in
+            guard let self = self else { return }
```

Per our style guide, the `return` should be on the next line. 
```
            guard let self = self else {
                return
            }
```

---

Also, I'm curious about the "silent failure" here. On the off chance there is no `self`, then nothing in the success block is called. Would it be good to call `failure` inside the `guard`'s `else` block?

---

Good catch! This has been addressed in https://github.com/wordpress-mobile/WordPress-iOS/pull/17333/commits/8c1f6d80d00606040f2df0caf8ea46c8145b4300.